.travis.yml

```diff
@@ -10,10 +11,13 @@ script:
 - tox
 - coverage run --source isatools -m py.test
 - coverage report -m
-branches:
-  only:
-    - master
-    - develop
+- tests/my-pip-uninstall matplotlib pandas numpy tox-travis
+- rm -rf .tox
+- PYTHONPATH="${PWD}" python tests/test_isatab_only_meta.py
+#branches:
+#  only:
```

Comment:
Commenting this was useful to test my own feature branch.  Feel free to uncomment.